src/auth-service/app.js

```diff
@@ -43,76 +43,69 @@ app.use(function (req, res, next) {
 });
 
 app.use(function (err, req, res, next) {
-  logger.error(`${constants.ENVIRONMENT} -- ${err.message}`);
   if (err.status === 404) {
+    logger.error(`this endpoint does not exist --- ${err.message}`);
```

Comment:
just making the auth logger notifs more descriptive for some runtime error cases.